### DISCUSSION TITLE
Support Intel GPU Hardware transcoding (Intel Quick Sync / VAAPI)

### DISCUSSION THREAD

**Discussion by rkkoszewski** [LizardByte/community#55] [Sunshine] [answered]
Sunshine does not seem to currently support Intel GPU transcoding and I could not find any discussion about it on this project. The latest Intel integrated GPUs are pretty capable, and they usually come with hardware transcoding support named Quick Sync.

As far as I'm aware, Sunshine is using FFMPEG for transcoding the video, and FFMPEG already supports Intel Quick Sync: https://trac.ffmpeg.org/wiki/Hardware/QuickSync

Is this something that could be added to Sunshine or is there something talking against it?

**Comment by ReenigneArcher** [comment]
See #77

**Comment by ReenigneArcher** [comment, marked as answer]
QSV encoding is now supported.
